src/backend/config.py

```diff
@@ -120,7 +120,7 @@ def get_ui_path() -> str:
     def post_process(key: str, value: str | None) -> str:
         if value is None or value == "/":
             return "/"
-        match = re.fullmatch(r"/?[\w./-]+(?:[\w]+)/?", value)
+        match = re.fullmatch(r"[\w/-]+[\w/]+", value)
```

Comment:
this still doesn't look right, because it will validate to true something like `/-///`. What about something like `/?\w[\w-]+(?:/[\w-]+/?)$` ?

Reply:
1) Is `/-///` not a valid path in this context?
2) Your regex allows something like `/fooobar-//`, which wouldn't be so great either.

I don't think we should be validating paths with regexes at all (ever tried to validate anything else remotely complex with regexes? Like an email address?).
But I couldn't find a library that can just tell me if something is a valid URL path.
What I'm trying to do with this commit is not excluding all possible invalid paths, but soften up the rules enough that valid paths don't fail validation anymore, as this will otherwise be a real problem with the Longhorn integration (see the error log in the linked issue)

Reply:
> 1. Is `/-///` not a valid path in this context?

From the test you provided, I'm assuming it's not. Generally speaking, it wouldn't be, but we also don't have a definition of what is expected. :shrug: 

>     2. Your regex allows something like `/fooobar-//`, which wouldn't be so great either.

True, it can certainly see some more thinking and a few more iterations.

> I don't think we should be validating paths with regexes at all (ever tried to validate anything else remotely complex with regexes? Like an email address?).

Well, yes. Regular expressions may be annoying, but they are incredibly powerful.

> What I'm trying to do with this commit is not excluding all possible invalid paths, but soften up the rules enough that valid paths don't fail validation anymore, as this will otherwise be a real problem with the Longhorn integration (see the error log in the linked issue)

What's the cutoff for how much we soften this validation? Because I'd be more comfortable removing it altogether, rather than have a validation that does a half-baked job. At least by not validating we know that we are not validating, rather than have the false sense of security from a validation check.

Reply:
> Because I'd be more comfortable removing it altogether, rather than have a validation that does a half-baked job

That's fine with me, I'll take the axe to the code then.

Reply:
Then, please, leave a note on the code saying that needs to be validated, and file an issue for it.